src/push/monitor.ts

```diff
@@ -148,6 +151,15 @@ export async function buildMonitorSchema(monitors: Monitor[], isV2: boolean) {
       monitor.setContent(content);
       Object.assign(schema, { content, filter });
     }
+    const size = monitor.size();
```

Comment:
`SIZE_LIMIT_KB` description specifies this is gzipped, whereas `monitor.size()` is not, maybe we should align the description

Reply:
It is though, the content extracted is the gzipped size of the monitor. Can you give me hint on where we should align it?